src/plugins/discover/public/services/discover_ebt_manager.ts

```diff
@@ -104,17 +132,40 @@ export class DiscoverEBTManager {
           },
         },
       });
+      core.analytics.registerEventType({
+        eventType: CONTEXTUAL_PROFILE_RESOLVED_EVENT_TYPE,
+        schema: {
+          [CONTEXTUAL_PROFILE_LEVEL]: {
+            type: 'keyword',
+            _meta: {
+              description:
+                'The profile level at which it was resolved i.e. rootLevel, dataSourceLevel, documentLevel',
```

Comment:
Would you suggest to rename the property too from `profileLevel` to `contextLevel`?

Reply:
Personally I would since I prefer the consistency/clarity and it's easier to align now than later, but it's also not a blocker IMO since it's for internal telemetry.

Reply:
Updated via https://github.com/elastic/kibana/pull/203124/commits/96d99de25ce8f5a8774aedb2932764b29048cd1a